tests/js/spec/components/smartSearchBar/utils.spec.tsx

```diff
@@ -58,3 +60,174 @@ describe('getLastTermIndex()', function () {
     expect(getLastTermIndex(query, 9)).toEqual(11);
   });
 });
+
+describe('getTagItemsFromKeys()', function () {
```

Comment:
Thanks for adding this test? This tests a good amount of the logic that you added, but I'd also like to see some additional component tests in the `index.spec.js` file to ensure that the component uses this function correctly. (unless those tests already exist and I've simply missed them)